core/iwasm/libraries/libc-builtin/libc_builtin_wrapper.c

```diff
@@ -1137,6 +1145,7 @@ static NativeSymbol native_symbols_libc_builtin[] = {
     REG_NATIVE_FUNC(sprintf, "($$*)i"),
     REG_NATIVE_FUNC(snprintf, "(*~$*)i"),
     REG_NATIVE_FUNC(puts, "($)i"),
+    REG_NATIVE_FUNC(clock, "()I"),
```

Comment:
There is another  API clock_gettime registered at line 1201, can you put them together?

Reply:
Done